backend/package.json

```diff
@@ -22,6 +22,7 @@
   },
   "homepage": "https://github.com/hhe-wtag/bidify#readme",
   "dependencies": {
+    "bcryptjs": "^2.4.3",
```

Comment:
We should use updated `bcrypt` version. Use this package instead please https://www.npmjs.com/package/bcrypt